.gitignore

```diff
@@ -39,3 +39,4 @@ gen.*.bat
 gen.*.sh
 .env
 scripts/auto_backtester/backtesting_*.csv
+.vscode
```

Comment:
I don't mind, but you could also add this to your global gitignore locally.

Reply:
Splitting them out kinda makes sense but why not a separate branch? Easier to cherry pick changes applicable to both?